src/rabbit_auth_backend_ldap.erl

```diff
@@ -208,13 +208,20 @@ evaluate0({equals, StringQuery1, StringQuery2}, Args, User, LDAP) ->
               evaluate(StringQuery1, Args, User, LDAP),
               evaluate(StringQuery2, Args, User, LDAP));
 
-evaluate0({match, StringQuery, REQuery}, Args, User, LDAP) ->
+evaluate0({match, {string, _} = StringQuery, {string, _} = REQuery}, Args, User, LDAP) ->
```

Comment:
The documentation states that strings can also be represented by plain strings which won't be included by this special case. Probably need to add another clause to check when `StringQuery` and `REQuery` are both lists.